internal/db/migrations/004_pubkeys.sql

```diff
@@ -3,7 +3,10 @@ CREATE TABLE pubkeys
   id BIGINT GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
   account_id BIGINT NOT NULL REFERENCES accounts(id),
   name TEXT NOT NULL CHECK (NOT empty(name)),
-  body TEXT NOT NULL CHECK (NOT empty(body))
+  body TEXT NOT NULL CHECK (NOT empty(body)),
+
+  UNIQUE(name, account_id),
```

Comment:
Should this instead be `UNIQUE(pubkey_resources, handle)`?

Reply:
IMO no.
we found out that in order to RunInstances we need the **keyName** - the name of the uploaded key. 
So in order to use the name we want to make sure it is unique.

Reply:
ok, I'm fine with that, I just thought handle is the name we use as AWS keyName, but I'm not sure if I got that correctly.

Reply:
no, the handle is the unique id AWS gives to a key - it looks something like `key-488030` (like the AMI identifier), the key name is saved in Pubkey 😃

Reply:
interesting... so it's really a name not an handle we need to pass? Interesting :thinking:
Ok then, ignore me! :+1:

Reply:
Not sure what you mean by this, pubkey must be unique in the main pubkeys table already, even before it gets deployed. In my working branch, there are two unique checks now: name, account + fingerprint, account.